operator/scheduler/model.go

```diff
@@ -317,38 +324,47 @@ func (s *SchedulerClient) updateModelStatus(ctx context.Context, model *v1alpha1
 		return err
 	}
 
+	if !existingModel.ObjectMeta.DeletionTimestamp.IsZero() {
+		// Don't update status on deleted objects
```

Comment:
The condition reflects that the object is in the process of being deleted rather than already deleted. The comment should reflect that. I'm in two minds about this change, because it means that if a delete gets stuck, we give even less information to the end-user about what is going on.